docs/src/rules/no-unused-vars.md

```diff
@@ -283,20 +283,24 @@ The `caughtErrors` option is used for `catch` block arguments validation.
 
 It has two settings:
 
-* `none` - do not check error objects. This is the default setting.
-* `all` - all named arguments must be used.
+* `all` - all named arguments must be used. This is the default setting.
+* `none` - do not check error objects.
 
-#### caughtErrors: none
+:::
```

Comment:
I think this `:::`  should be moved to the end of the `#### caughtErrors: none` section where it's missing.